src/main/java/edu/wisc/my/personalizedredirection/service/parser/RedirectUrlCSVParser.java

```diff
@@ -33,7 +36,8 @@ public AttributeMapList parseResource(Resource resource) throws PersonalizedRedi
 			br.close();
 
 		} catch (Exception e) {
-			throw new PersonalizedRedirectionException(e.getMessage());
+			logger.error(e.getMessage());
+			throw new RuntimeException(e.getMessage());
```

Comment:
Consider chaining the exception on the wrap and throw new, so as to not lose stack trace context that conceivably might be helpful.